manifest-build-tools/application/update_rackhd_version.py

```diff
@@ -0,0 +1,207 @@
+#!/usr/bin/env python
+
+"""
+usage:
+./on-tools/manifest-build-tools/HWIMO-BUILD on-tools/manifest-build-tools/application/update_rackhd_version.py \
+--manifest-repo-dir build-manifests \
+--builddir b \
+--git-credential https://github.com/PengTian0,GITHUB \
+                     
+The required parameters:
+manifest-repo-dir: The directory of manifest repository
+builddir: The destination for repositories stored.
```

Comment:
for "builddir" :  which repositories ? I believe RackHD and on-xxx should be there. please make it more clear.